src/config/kyuubi.py

```diff
@@ -65,9 +72,24 @@ def _ha_conf(self) -> dict[str, str]:
             "kyuubi.ha.zookeeper.auth.digest": self._get_zookeeper_auth_digest(),
         }
 
+    @property
+    def _tls_conf(self) -> dict[str, str]:
```

Comment:
**nitpick**: Related to previous comment, this type annotation is incompatible with 3.8. No more typing nitpicks, promise!